### PR TITLE
google workspace app

### PR DESCRIPTION
**Pull request by MariePMcGarry**
## Purpose of this pull request

See https://jasklabs.atlassian.net/browse/CON-1133 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/google/workspace/collect-logs-legacy.md [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+
+### Alert types
+
+All the alerts are in JSON format. Most of the alerts have few common fields like alertId, customerId, createTime, source, type and data. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information about different alert types refer this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
+
+
+### Collection overview
+
+Sumo Logic provides a serverless solution which pulls logs from Google Workspace with API calls. You can configure the list alerts to be collected, but by default all alerts are collected. The alerts  are then forwarded to Sumo Logic’s HTTP endpoint in JSON format. By default the collection starts from the current date and time, but this setting is also configurable as detailed in the [Advanced configuration](#Configure_Collection_for_Google_Workspace_Alert_Center#Advanced_configuration) section.
```
Comment (kimsauce) [inline docs/integrations/google/workspace/collect-logs-legacy.md:27]:
```suggestion
Sumo Logic provides a serverless solution which pulls logs from Google Workspace with API calls. You can configure the list alerts to be collected, but by default all alerts are collected. The alerts  are then forwarded to Sumo Logic’s HTTP endpoint in JSON format. By default the collection starts from the current date and time, but this setting is also configurable as detailed in the [Advanced configuration](#Advanced-configuration) section.
```

File: docs/integrations/google/workspace/collect-logs-legacy.md [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+   * `region`: The Region where the Google function will be deployed. For example: "us-central1"
+   * `project_id`: The project id of the project where the collector and all its resources will be deployed
+   * `delegated_email`: The valid email address of one of your org's G Suite super admin users.
+   * `Sumo_endpoint`: The Sumo Logic HTTP endpoint created in Step 1
+4. Run the following script:
+  ```bash
+  sh sumo_gsuite_alerts_collector_deploy.sh
+  ```
+5. In the command prompt, enter `N` for the following question: `Allow unauthenticated invocations of new function`.
+6. Copy the **Client ID** displayed at the end of the script output. You will use the Client Name field when you [configure Google Workspace Alert Center to allow client API access](#Configure_Google_Cloud_Platform_Collection_for_Google_Workspace_Alert_Center#configure-google-workspace-alert-center-to-allow-client-api-acce) in the following task.
```
Comment (kimsauce) [inline docs/integrations/google/workspace/collect-logs-legacy.md:135]:
```suggestion
6. Copy the **Client ID** displayed at the end of the script output. You will use the Client Name field when you [configure Google Workspace Alert Center to allow client API access](#configure-google-workspace-alert-center-to-allow-client-api-acce) in the following task.
```

File: docs/integrations/google/workspace/collect-logs-legacy.md [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+The script you used in this procedure configured the environment variables in the config file. You can update the variables by following the instructions Google's [Updating environment variables documentation](https://cloud.google.com/functions/docs/env-var).
+:::
+
+### Configure Google Workspace Alert Center to allow client API access
+
+This section explains how to configure Google Workspace Alert Center to allow API access.
+
+Follow the step 2 and step 3 under “Set up the Alert Center API” [docs](https://developers.google.com/admin-sdk/alertcenter/guides/prerequisites) to enable alert center API and grant domain-wide access to the application.
+
+If you're using the [Configure Google Cloud Platform Collection for Google Workspace Alert Center](#Configure_Google_Cloud_Platform_Collection_for_Google_Workspace_Alert_Center), use the **Client ID** for the service account copied in Step 6 of the following [docs](#Configure_Google_Cloud_Platform_Collection_for_Google_Workspace_Alert_Center#google-cloud-platform-gcp-%C2%A0collection).
```
Comment (kimsauce) [inline docs/integrations/google/workspace/collect-logs-legacy.md:148]: What should that last link be pointing to?

File: docs/integrations/google/workspace/collect-logs-legacy.md [hunk 4]
```diff
@@ -0,0 +1,504 @@
...
+
+To configure Google Workspace Alert Center:
+1. Go to your G Suite domain's Admin console (see instructions on [signing in to your Admin console](https://support.google.com/a/answer/182076)), go to **Security > Access and data control > API Controls**.<br/><img src={useBaseUrl('img/integrations/google/gw.png')} alt="google workspace" width="200"/>
+1. In the newly opened window, click **Manage Domain-wide Delegation **at the bottom.<br/><img src={useBaseUrl('img/integrations/google/GSuite_Step3.png')} alt="google workspace" />
+1. Click **Add new** button on the top.<br/><img src={useBaseUrl('img/integrations/google/GSuite_Step4.png')} alt="google workspace" />
+1. Enter the **Client ID** for the service account copied in Step 2, then in the **OAuth Scopes** field enter the following: [https://www.googleapis.com/auth/apps.alerts ](https://www.googleapis.com/auth/apps.alerts). <br/><img src={useBaseUrl('img/integrations/google/GSuite_Step5.png')} alt="google workspace" width="400"/>
```
Comment (kimsauce) [inline docs/integrations/google/workspace/collect-logs-legacy.md:156]:
```suggestion
1. Enter the **Client ID** for the service account copied in Step 2, then in the **OAuth Scopes** field enter the following: [https://www.googleapis.com/auth/apps.alerts](https://www.googleapis.com/auth/apps.alerts). <br/><img src={useBaseUrl('img/integrations/google/GSuite_Step5.png')} alt="google workspace" width="400"/>
```

File: docs/integrations/google/workspace/collect-logs.md [hunk 5]
```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
...
     * Google Drive
     * Google Login
     * Google Token
 
 ### About Source Configuration  
 
-Currently, the source name for Google Workspace is still **G Suite Apps Audit Source**, which will be changed/updated shortly.
-
-Configure one [G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source) for each Google App from which you want to collect events:
+Configure one [Google Workspace Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md) for each Google App from which you want to collect events:
```
Comment (kimsauce) [inline docs/integrations/google/workspace/collect-logs.md:33]:
```suggestion
Configure one [Google Workspace Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source) for each Google App from which you want to collect events:
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
some suggestions

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi @MariePMcGarry , added some more changes and did a slight reorg.

**Comment by kimsauce** [comment]
@MariePMcGarry , edited your PR and obtained Mohit's approval (see https://jasklabs.atlassian.net/browse/CON-1133).

@stacykor , will merge this.

File: docs/integrations/google/workspace/collect-logs.md [hunk 6]
```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
 **Google Workspace Alert Center** alerts are in JSON format. Most of the alerts have a few common fields. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information, see this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
 
 
-## Configure Collection for Google Workspace Audit Source
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:21]: Actually we need to keep "Audit" as we are using the "G Suite Apps Audit" source, not the "Google Workspace" source. The latter only collects user data and it's for CSE mostly (I know it's confusing).

File: docs/integrations/google/workspace/collect-logs.md [hunk 7]
```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
...
+## Configure Collection for Google Workspace Source
 
-This section provides instructions for configuring log collection for Google Workspace with Audit Source.
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:23]: Same comment as above

File: docs/integrations/google/workspace/collect-logs.md [hunk 8]
```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
...
+This section provides instructions for configuring log collection for Google Workspace Source.
 
-[Google Workspace](#configure-collection-for-google-workspace-audit-source): Monitors and analyzes the activity across all the Google Workspace Apps in one place. You can configure collection for each Google App for which you want to analyze events:
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:25]: Same comment

File: docs/integrations/google/workspace/collect-logs.md [hunk 9]
```diff
@@ -18,21 +18,19 @@ This procedure explains how to collect logs from Google Workspace and ingest the
...
     * Google Admin
     * Google Drive
     * Google Login
     * Google Token
 
 ### About Source Configuration  
 
-Currently, the source name for Google Workspace is still **G Suite Apps Audit Source**, which will be changed/updated shortly.
-
-Configure one [G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source) for each Google App from which you want to collect events:
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:35]: I don't know when we will rename this source ( to "Google Workspace Audit Source"?), right now it's shown as "G Suite Apps Audit Source"

File: docs/integrations/google/workspace/collect-logs.md [hunk 10]
```diff
@@ -44,7 +42,7 @@ Google Workspace Drive Audit events are only logged for files owned by users wit
 
 When you configure your Source Categories, you can configure and use them in two different ways.
 
-**One Single Source Category for all Sources.** For users who are setting up the Google Workspace Apps Audit Source for the first time, we suggest that you use the same single Source Category for each Google Apps Audit Source. For example, **google_apps**.
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:47]: "Apps Audit" is  actually correct

File: docs/integrations/google/workspace/collect-logs.md [hunk 11]
```diff
@@ -53,7 +51,7 @@ When you configure your Source Categories, you can configure and use them in two
 * google_app_login
 * Google_app_token
 
-A Google Workspace Apps Audit Source uses the [Google Apps Reports API](https://developers.google.com/admin-sdk/reports/v1/get-start/getting-started) to ingest all audit logs via watchpoints. Activity from the following Google apps are supported in Sumo's Google Workspace App:
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:56]: The old text is actually correct

File: docs/integrations/google/workspace/collect-logs.md [hunk 12]
```diff
@@ -67,28 +65,28 @@ Only one Source should be configured per app. In other words, you might set up o
 
 This Source uses OAuth to integrate with the Google Apps Reports API. Therefore, your Google Apps credentials are never stored by Sumo Logic, and we have no visibility into the details of your Google Apps account.  Sumo Logic only stores OAuth tokens that are generated after authentication and authorization.
 
-When creating or modifying a Google Apps Audit Source, you will be required to authenticate with Google using the credentials of a user that has access rights to the account, and to the Reports API. See Google's [Reports API: Prerequisites](https://developers.google.com/admin-sdk/reports/v1/guides/prerequisites) documentation for more details. During Google's OAuth consent flow, you will also be asked to grant the Sumo Logic app permission to use the Reports API.
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:70]: All the old texts on "Apps Audit" (and below) are actually correct.

File: docs/integrations/google/workspace/collect-logs.md [hunk 13]
```diff
@@ -200,496 +198,3 @@ _sourceCategory=google*
 | count by email
 | sort by _count desc, email asc
 ```
```
Comment (duchatran) [inline docs/integrations/google/workspace/collect-logs.md:200]: This removal is correct. We just need to point to the new C2C source page, currently at https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-alertcenter/

**Pull request merged**